apps/yapms/src/lib/components/modals/sharemodal/ShareModal.svelte

```diff
@@ -43,6 +44,12 @@
 		}
 	}
 
+	function loadTCT() {
+		if (tctFiles && tctFiles.length > 0) {
+			loadFromTCTFile(tctFiles);
```

Comment:
On success this should close the share modal like opening a normal file does.

Reply:
Added